en/restore-data-using-tidb-lightning.md

```diff
@@ -96,68 +98,102 @@ TiDB Lightning Helm chart supports both local and remote data sources.
+    2. Public Cloud Account Grant Permissions
 
         There are three methods to grant permissions. The configuration varies with different methods. For details, see [Backup the TiDB Cluster on AWS using BR](backup-to-aws-s3-using-br.md#three-methods-to-grant-aws-account-permissions).
 
-        * If you grant permissions by importing Amazon S3 AccessKey and SecretKey, or if you use Ceph or GCS as the storage, use the following configuration:
-
-            {{< copyable "" >}}
-
-            ```yaml
-            apiVersion: v1
-            kind: Secret
-            metadata:
-              name: cloud-storage-secret
-            type: Opaque
-            stringData:
-              rclone.conf: |
-                [s3]
-                type = s3
-                provider = AWS
-                env_auth = false
-                access_key_id = ${access_key}
-                secret_access_key = ${secret_key}
-                region = us-east-1
-
-                [ceph]
-                type = s3
-                provider = Ceph
-                env_auth = false
-                access_key_id = ${access_key}
-                secret_access_key = ${secret_key}
-                endpoint = ${endpoint}
-                region = :default-placement
-
-                [gcs]
-                type = google cloud storage
-                # The service account must include Storage Object Viewer role
-                # The content can be retrieved by `cat ${service-account-file} | jq -c .`
-                service_account_credentials = ${service_account_json_file_content}
-            ```
-
-        * If you grant permissions by associating Amazon S3 IAM with Pod or with ServiceAccount, you can ignore `s3.access_key_id` and `s3.secret_access_key`:
-
-            {{< copyable "" >}}
-    
-            ```yaml
-            apiVersion: v1
-            kind: Secret
-            metadata:
-              name: cloud-storage-secret
-            type: Opaque
-            stringData:
-              rclone.conf: |
-                [s3]
-                type = s3
-                provider = AWS
-                env_auth = true
-                access_key_id =
-                secret_access_key =
-                region = us-east-1
-            ```
-
-            Fill in the placeholders with your configurations and save it as `secret.yaml`, and then create the `Secret` via `kubectl apply -f secret.yaml -n ${namespace}`.
+        * Grant permissions by importing AccessKey and SecretKey
+
+            1. Download [backup-rbac.yaml](https://github.com/pingcap/tidb-operator/blob/master/manifests/backup/backup-rbac.yaml), and execute the following command to create the role-based access control (RBAC) resources in the `${namespace}` namespace:
```

Comment:
Do you need to change the yaml file version for v1.1 docs? If you do, please remember to edit the link after the PR is cherry-picked to v1.1 branch.

Reply:
Thanks for reminding, this link in v1.1 and dev is the same, and we don't need to change it.